### PR TITLE
Add assimulo

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/assimulo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  imports:
+    - assimulo
+
+  requires:
+      - nose
+about:
+  home: http://www.jmodelica.org/assimulo
+  license: GPLv3, LGPLv3, LGPLv2
```
Comment (scopatz) [inline recipes/assimulo/meta.yaml]: Can you add the `license_file`, please?

File: recipes/assimulo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - cython
+    - sundials
+    - gcc  # [unix]
+  run:
+    - python
+    - numpy x.x
```
Comment (scopatz) [inline recipes/assimulo/meta.yaml]: Also, it isn't clear to me, but I think we might not be pinning numpy like this anymore.

File: recipes/assimulo/meta.yaml (hunk 1 above)
Comment (jschueller) [inline recipes/assimulo/meta.yaml]: @scopatz, they do not provide any license in their archive:
https://pypi.python.org/packages/4c/c0/19a54949817204313efff9f83f1e4a247edebed0a1cc5a317a95d3f374ae/Assimulo-2.9.zip

File: recipes/assimulo/meta.yaml (hunk 2 above)
Comment (jschueller) [inline recipes/assimulo/meta.yaml]: @jakirkham told me this

File: recipes/assimulo/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/assimulo/meta.yaml]: Could we ask them to start including a license file in their `MANIFEST.in`?

File: recipes/assimulo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  - vc10  # [win and py34]
+  - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - cython
+    - sundials
+    - gcc  # [unix]
```
Comment (jakirkham) [inline recipes/assimulo/meta.yaml]: What is this being used for?

File: recipes/assimulo/meta.yaml (hunk 1 above)
Comment (jschueller) [inline recipes/assimulo/meta.yaml]: Yes, I sent a mail to the maintainer of assimulo a few days ago, no answer yet.

File: recipes/assimulo/meta.yaml (hunk 2 above)
Comment (scopatz) [inline recipes/assimulo/meta.yaml]: Ok thanks, @jakirkham

File: recipes/assimulo/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  requires:
+      - nose
+about:
+  home: http://www.jmodelica.org/assimulo
+  license: GPLv3, LGPLv3, LGPLv2
+  summary: A package for solving ordinary differential equations and differential algebraic equations.
+
+extra:
+  recipe-maintainers:
+    - jschueller
```
Comment (jschueller) [inline recipes/assimulo/meta.yaml]: Yes, I expect him to react on this too, but he doesn't seem to have a github account though

Comment (jschueller) [inline recipes/assimulo/meta.yaml]: He replied, I added him.

File: recipes/assimulo/meta.yaml (hunk 1 above)
Comment (jschueller) [inline recipes/assimulo/meta.yaml]: The maintainer replied and will fix it for the next version!

Comment (jschueller) [inline recipes/assimulo/meta.yaml]: @scopatz I think this is ready to go now.

**Comment by scopatz** [comment]
Thanks!

**Comment by jschueller** [comment]
thanks, now I can rerun #1727

**Pull request merged**